Components/Input/input.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
can add meta tags to give additional site information

Reply:
Thanks for the suggestion would surely do that once i complete everything else. Due to college work i am under a time crunch.